WordPress/Classes/ViewRelated/Blog/Blog Dashboard/Cards/Posts/BlogDashboardCardFrameView.swift

```diff
@@ -0,0 +1,115 @@
+import UIKit
+import Gridicons
+
+/// A view that consists of the frame of a Dashboard card
+/// Title, icon and action can be customizable
+class BlogDashboardCardFrameView: UIView {
+
+    /// The main stack view, in which the header and the content
+    /// are appended to.
+    private lazy var mainStackView: UIStackView = {
+        let stackView = UIStackView()
+        stackView.axis = .vertical
+        stackView.translatesAutoresizingMaskIntoConstraints = false
+        return stackView
+    }()
+
+    /// Header in which icon, title and chevron are added
+    private lazy var headerStackView: UIStackView = {
+        let topStackView = UIStackView()
```

Comment:
What do you think of adding the arranged subviews here instead of down below in init?
```suggestion
        let topStackView = UIStackView(arrangedSubviews:[
            iconImageView,
            titleLabel,
            chevronImageView
        ])
```

Reply:
Same as above!